sale_warn_option/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2024 Moduon Team S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0)
+
+{
+    "name": "Sale Warn Option",
+    "summary": "Add Options to Sale Warn Messages",
+    "version": "15.0.1.0.0",
+    "development_status": "Alpha",
+    "category": "Sales/Sales",
+    "website": "https://github.com/OCA/sale-workflow",
+    "author": "Moduon, Odoo Community Association (OCA)",
+    "maintainers": ["Shide"],
```

Comment:
```suggestion
    "maintainers": ["Shide", "rafaelbn"],
```

See [our guidelines](https://doc.moduon.team/doc/estandares-de-contribucion-gxxTTBnNnb#h-mantenedores).